### ISSUE TITLE
Cadastro de Usuario

### ISSUE THREAD

**Issue by maurodev16** [maurodev16/Tierder#2]
1. id = Primarykey
2. username = String
3. email = String
4. password = String
5. token = String
6. isVerified = String
7. animals = List[]
8. createAt
9. updateAt 
10. Campos basico ate agora.

**Comment by Lazaro-manuel** [comment]
Vou começar por este hoje.

**Comment by maurodev16** [comment]
> Vou começar por este hoje.

OK, mais acho que a lista de animais não deveria estar ali no cadastro de usuario, isso só deveria aparecer na "perquisar usuários e animais"

**Comment by Lazaro-manuel** [comment]
> 

Verdade, não tinha visto. A lista é uma endpoint diferente de listar animais.

**Comment by maurodev16** [comment]
> > 
> 
> Verdade, não tinha visto. A lista é uma endpoint diferente de listar animais.

Exato! Porém o animal tem que estar atrelado a lista

**Comment by Lazaro-manuel** [comment]
> 

Blz mano...

**Comment by Lazaro-manuel** [comment]
Terminei este modulo vou começar o outroo!

**Comment by Lazaro-manuel** [comment]
Vou precisar das suas credências na aws pra fazer o deploy, ou posso fazer no heruku
